src/core/src/UsdUtil.cpp

```diff
@@ -360,6 +367,27 @@ pxr::CesiumData getOrCreateCesiumData() {
     return defineCesiumData(CesiumDataPath);
 }
 
+pxr::SdfPath getPathToCurrentIonServer() {
+    return pxr::SdfPath("/CesiumServers/IonOfficial");
+}
+
+pxr::CesiumIonServer getOrCreateCurrentIonServer() {
+    const auto currentIonServerPath = getPathToCurrentIonServer();
+
+    if (isCesiumIonServer(currentIonServerPath)) {
+        auto stage = getUsdStage();
+        auto currentIonServer = pxr::CesiumIonServer::Get(stage, currentIonServerPath);
+        return currentIonServer;
+    }
+
+    return defineCesiumIonServer(currentIonServerPath);
+}
+
+[[maybe_unused]] std::vector<pxr::CesiumIonServer> getAllIonServerPrims() {
```

Comment:
I'll be using this function very soon. It wasn't necessary for this piece.

Reply:
I know you're not using VS Code, but do you know if there's a way in it to check for functions with no references? CLion might due to the fact you added [[maybe_unused]].